src/pages/SignMessage/components/SignSuccess.tsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import { faCheck } from '@fortawesome/free-solid-svg-icons';
+import { SignedMessageInfoType } from '@multiversx/sdk-dapp/reduxStore/slices';
+import { PageState } from '@multiversx/sdk-dapp/UI';
+import { CopyButton } from '@multiversx/sdk-dapp/UI/CopyButton';
+import { Link } from 'react-router-dom';
+import { routeNames } from 'routes';
+import { SignMessageResultPropsType } from '../types';
+
+export const SignSuccess = ({
+  message,
+  onClear
```

Comment:
I think it would be easyer to use onClear directly from sdk-dapp to show that it's generic